src/actions/channels.js

```diff
@@ -968,6 +968,20 @@ export function getChannelStats(channelId: string): ActionFunc {
     };
 }
 
+export function getChannelMembersEmails(channelId: string): ActionFunc {
+    return async (dispatch, getState) => {
+        let result;
+        try {
+            result = await Client4.getChannelMembersEmails(channelId);
+        } catch (error) {
+            forceLogoutIfNecessary(error, dispatch, getState);
```

Comment:
do we need force logout of users on failure cases?

Reply:
All other catch blocks were doing the same thing like getChannelStats method etc..so I followed the same and used in my code.